Runner.php

```diff
@@ -20,24 +20,47 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
+        if(isset($this->config['include_paths'])) {
+            $this->queueWithIncludePaths();
+        } else {
+            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
+        }
+
+        $this->server->process_work(false);
+    }
+
+    public function queueWithIncludePaths() {
+        foreach ($this->config['include_paths'] as $f) {
+            if ($f !== '.' and $f !== '..') {
```

Comment:
While you're in here -- do you want to fix https://github.com/codeclimate/codeclimate-phpcodesniffer/issues/14 for this engine?